projects/packages/masterbar/src/admin-color-schemes/colors/fresh/colors.scss

```diff
@@ -4,6 +4,13 @@
 @import "../_inline-help";
 @import "sidebar-notice";
 
+#wpadminbar .ab-icon,
+#wpadminbar .ab-icon:before,
```

Comment:
Can we only have `#wpadminbar .ab-icon` here, I think the other `*:before` and `*:after` are already there from Core 🤔

Reply:
Ok, I'll update it